src/screens/OnboardingScreen/OnboardingScreen.js

```diff
@@ -1,88 +1,114 @@
+/* eslint-disable react-native/no-inline-styles */
 import React from 'react';
-import { View, Text, Image, TouchableOpacity } from 'react-native';
+import {View, Text, Image, TouchableOpacity} from 'react-native';
 import Onboarding from 'react-native-onboarding-swiper';
 import styles from './styles';
 
 const Dots = ({selected}) => {
-    let backgroundColor;
-    backgroundColor = selected ? '#51AD28' : 'rgba(0, 0, 0, 0.3)';
+  let backgroundColor;
+  backgroundColor = selected ? '#51AD28' : 'rgba(0, 0, 0, 0.3)';
```

Comment:
Is is possible to change the color codes in to one method? either # or rgb and not both in the same line.

Reply:
Done